README.md

```diff
@@ -0,0 +1,49 @@
+# EQ Load Generator
+
+This folder contains the config for the EQ load generator infrastructure. It uses the eq-survey-runner-benchmark repo in order to configure a load test using http://locust.io.
+
+* Terraform is used to create a network and K8s cluster in GCP (this defaults to a single machine)
+* We then apply some K8 config to the cluster
+* We then download eq-survey-runner-benchmark and run its k8s config.
+
+## Prerequisites
+
+1. Install [Terraform Version Manager](https://github.com/kamatama41/tfenv)
+
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/docs/quickstarts)
+
+1. Login to your [google cloud console](https://console.cloud.google.com/), make sure you are using the correct account and accept any Terms of Service.
+
+1. Login to the CLI with your Google account `gcloud auth login` and `gcloud auth application-default login`
+
+1. Install kubectl with `gcloud components install kubectl`
+
+1. Install Helm with `brew install kubernetes-helm` and then run `helm init --client-only`
+
+1. Install Helm Tiller plugin for tillerless deploys `helm plugin install https://github.com/rimusz/helm-tiller`
+
+## Creating an environment
+
+### Development
+
+Use an ENV name which follows the naming conventions here: https://cloud.google.com/storage/docs/naming
+
+Rename `terraform.tfvars.example` to `terraform.tfvars` and fill in the values. (Ask a team member for help).
+
+Create with `ENV=your-env RUNNER_URL=https://your-runner.gcp.dev.eq.ons.digital ./create.sh`
+
+Destroy with `ENV=your-env ./destroy.sh`
+
+If you are deploying to an existing environment, you can use the `EXISTING_PROJECT_ID` variable to define this:
+```
+EXISTING_PROJECT_ID=existing-project ENV=existing-project RUNNER_URL=https://your-runner.gcp.dev.eq.ons.digital ./create.sh
```

Comment:
I know this is how it's done in the other Terraform, but it would make more sense for `EXISTING_PROJECT_ID` to become a boolean `IMPORT_EXISTING_PROJECT` and to use the environment name from `ENV`. That way we don't have this confusing duplication of the project name across two environment variables. This command would become:

```
IMPORT_EXISTING_PROJECT=True ENV=existing-project RUNNER_URL=https://your-runner.gcp.dev.eq.ons.digital ./create.sh
```

Reply:
I've had a chat about this with @pricem14pc @MebinAbraham and @JamesGardiner and also think it would be clearer if we renamed `ENV` to `PROJECT_NAME`.